frontend/public/components/masthead-toolbar.jsx

```diff
@@ -160,6 +161,16 @@ class MastheadToolbarContents_ extends React.Component {
     history.push('/settings/cluster');
   }
 
+  _onNotificationBadgeClick() {
+    if (window.innerWidth >= BREAKPOINTLG) {
```

Comment:
We should avoid having this logic in multiple places. Is there a `onHide`/`onShow` callback we can set in the notification drawer.